src/script_chainer/services/process_manager.py

```
@@ -243,6 +248,11 @@ def open_process(
 
         self.process = subprocess.Popen(command, **popen_kwargs)
 
+        # 将子进程加入 Job Object，父进程退出时自动清理
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)